### PR TITLE
Add helper to fetch details from etcd

### PR DESCRIPTION
**Pull request by anmolbabu**
tendrl-bug-id: Tendrl/alerting#10,Tendrl/alerting#11,Tendrl/alerting#12,Tendrl/alerting#14
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@Tendrl/tendrl-core This is a PR split out from https://github.com/Tendrl/alerting/pull/15. Please review

File: tendrl/alerting/etcd/manager.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+from tendrl.alerting.persistence.tendrl_definitions \
+    import TendrlDefinitions
+from tendrl.common.config import ConfigNotFound
+from tendrl.common.config import TendrlConfig
+from tendrl.common.etcdobj.etcdobj import Server as etcd_server
+from tendrl.common.singleton import to_singleton
+import time
+import yaml
+
+config = TendrlConfig()
```
Comment (shtripat) [inline tendrl/alerting/etcd/manager.py]: Change as per latest refactoring changes.

Comment (anmolbabu) [inline tendrl/alerting/etcd/manager.py]: Ok

Comment (anmolbabu) [inline tendrl/alerting/etcd/manager.py]: ok

File: tendrl/alerting/etcd/manager.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            etcd.EtcdException,
+            TypeError
+        ) as ex:
+            LOG.error(
+                'Failed to fetch supported alert types',
+                exc_info=True
+            )
+            raise EtcdError(str(ex))
+
+    def load_defs(self):
```
Comment (r0h4n) [inline tendrl/alerting/etcd/manager.py]: change func name to "update_defs"

File: tendrl/alerting/etcd/manager.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            SyntaxError,
+            TypeError
+        ) as ex:
+            LOG.error(
+                'Failed to fetch alerts. Error %s' % str(ex),
+                exc_info=True
+            )
+            raise EtcdError(str(ex))
+        for child in alerts._children:
+            alerts_arr.append(json.loads(child['value']))
```
Comment (r0h4n) [inline tendrl/alerting/etcd/manager.py]: why not filter in this for loop itself? you are basically iterating through the same data set twice (L82)

Comment (anmolbabu) [inline tendrl/alerting/etcd/manager.py]: Done

File: tendrl/alerting/etcd/manager.py (hunk 2 above)
Comment (anmolbabu) [inline tendrl/alerting/etcd/manager.py]: Done

**Comment by anmolbabu** [comment]
Test for persister is part of https://github.com/Tendrl/alerting/pull/22

**Pull request merged**